homeassistant/components/evohome/climate.py

```diff
@@ -342,16 +387,22 @@ async def async_tcs_svc_request(self, service: str, data: dict[str, Any]) -> Non
 
         await self._set_tcs_mode(mode, until=until)
 
-    async def _set_tcs_mode(self, mode: str, until: dt | None = None) -> None:
+    async def _set_tcs_mode(self, mode: str, until: datetime | None = None) -> None:
         """Set a Controller to any of its native EVO_* operating modes."""
+
         until = dt_util.as_utc(until) if until else None
+
         await self._evo_broker.call_client_api(
-            self._evo_tcs.set_mode(mode, until=until)
+            self._evo_tcs.set_mode(mode, until=until)  # type: ignore[arg-type]
```

Comment:
Not why you've reverted this ignore.

Reply:
I put it back in because the pre-commit insisted. 

I fetched the latest HA, and merged it, then re-ran script/setup.

It accepted the commit, and when I pushed it, the CI passed.

Sorry: I don't know what else to do!